### PR TITLE
Update cloud tests to work with viridian API-1785

### PR DESCRIPTION
**Pull request by srknzl**
RC PR https://github.com/hazelcast/hazelcast-remote-controller/pull/56

If we can merge the RC PR, the 0.8-snapshot.jar will be updated, ad I can test the workflows on my fork. I will use my account's API key and secrets. I will merge this PR to my fork's master.

### CODE REVIEW

File: HazelcastCloudTests/javahazelcastcloudtests/src/test/java/HazelcastCloudManager.java [hunk 1]
```diff
@@ -16,22 +16,25 @@
 import java.util.UUID;
 import java.util.concurrent.TimeUnit;
 
-public class HazelcastCloudManager {
+public class CloudManager {
```
Comment (mdumandag) [inline HazelcastCloudTests/javahazelcastcloudtests/src/test/java/HazelcastCloudManager.java:19]: So, we duplicate the code in the remote controller here? Can't we re-use it?

**Review by mdumandag — COMMENTED** [review]
Reviewed for Python and Java clients

File: HazelcastCloudTests/javahazelcastcloudtests/src/test/java/HazelcastCloudManager.java (hunk 1 above)
Comment (srknzl) [reply HazelcastCloudTests/javahazelcastcloudtests/src/test/java/HazelcastCloudManager.java:19]: 🤔 How? Do you propose to download it?

Comment (mdumandag) [reply HazelcastCloudTests/javahazelcastcloudtests/src/test/java/HazelcastCloudManager.java:19]: No, I meant adding a dependency to the remote controller in pom.xml

File: .github/workflows/cloud_java_client_test.yaml [hunk 2]
```diff
@@ -27,7 +27,7 @@ jobs:
         with:
           java-version: 8
           
-      - name: Run cloud tests
+      - name: Run cloud tests for UAT
```
Comment (mdumandag) [inline .github/workflows/cloud_java_client_test.yaml:30]: Will this work with the recent changes on the remote controller? I mean we made some changes for the viridian, so I am wondering if they apply to UAT as well?

File: .github/workflows/cloud_python_client_test.yaml [hunk 3]
```diff
@@ -71,19 +71,29 @@ jobs:
           API_KEY: ${{ secrets.APIKEY_PROD }}
           API_SECRET: ${{ secrets.APISECRET_PROD }}
         run: |
-          python start_remote_controller.py --use-simple-server &> rc_standard_cluster.log &
+          python start_remote_controller.py --use-simple-server &> rc.log &
```
Comment (mdumandag) [inline .github/workflows/cloud_python_client_test.yaml:74]: Same as above, will this work with the recent changes in the remote controller?

File: .github/workflows/cloud_java_client_test.yaml (hunk 2 above)
Comment (srknzl) [reply .github/workflows/cloud_java_client_test.yaml]: removed all uat and prod start rc steps

**Review by mdumandag — APPROVED** [review]
Looks OK for Python and Java

**Comment by srknzl** [comment]
The PR will be kept on hold until @emreyigit  does the migration of .net cloud testing code to .net client repo. Currently I cannot get it building.

**Review by mdumandag — APPROVED** [review]

**Review by emreyigit — APPROVED** [review]
Looks good tome.

**Pull request merged**